test/build.bats

```diff
@@ -338,7 +338,29 @@ DEF
   assert_build_log <<OUT
 ruby-2.0.0: --prefix=$INSTALL_ROOT
 make -j 2
-make install DOGE="such wow"
+make install DESTDIR= DOGE="such wow"
+OUT
+}
+
+@test "setting MAKE_INSTALL_DESTDIR" {
+  cached_tarball "ruby-2.0.0"
+
+  stub uname '-s : echo Linux'
+  stub_make_install
+
+  export MAKE_INSTALL_DESTDIR='/tmp/foo'
+  run_inline_definition <<DEF
+install_package "ruby-2.0.0" "http://ruby-lang.org/ruby/2.0/ruby-2.0.0.tar.gz"
```

Comment:
I'm running into an issue building 2.7.1 that has to do with the fact that some libs are installed into x.y.0 instead of x.y.z; see https://github.com/rbenv/ruby-build/pull/1416#issuecomment-619133360 for details.

I think it might be good to test with e.g. 2.0.1 or 2.1.2 or something that doesn't have the minor and patch set to 0, just to exercise that use case.